app/experimenter/nimbus-ui/src/components/PageSummary/index.test.tsx

```diff
@@ -119,27 +112,6 @@ describe("PageSummary", () => {
     );
   });
 
-  it("can cancel review when the experiment is in the review state", async () => {
```

Comment:
same here

Reply:
oh I think this test case is not duplicated

Reply:
it's a little hard to find (so many tests for this stuff in so many places) -- https://github.com/mozilla/experimenter/blob/9a8f02df68452fd3a464914ae1f282bf681abf08/app/experimenter/nimbus-ui/src/components/Summary/index.test.tsx#L143